ghedesigner/radial_numerical_borehole.py

```diff
@@ -373,67 +332,30 @@ def fill_f2(fx_2, cell):
 
             # For the idx == 0 case:
 
-            _b[0] = -radial_cell[previous_temperature_idx, 0] - heat_flux / ad
+            _b[0] = -radial_cell[CellProps.TEMP, 0] - heat_flux / ad
 
             # For the idx == n-1 case
 
             _dl[self.num_cells - 2] = 0.0
             _d[self.num_cells - 1] = 1.0
-            _b[self.num_cells - 1] = radial_cell[previous_temperature_idx, self.num_cells - 1]
+            _b[self.num_cells - 1] = radial_cell[CellProps.TEMP, self.num_cells - 1]
 
             # Now handle the 1 to n-2 cases with numpy slicing and vectorization
-            _b[1: self.num_cells - 1] = -radial_cell[previous_temperature_idx, 1: self.num_cells - 1]
+            _b[1: self.num_cells - 1] = -radial_cell[CellProps.TEMP, 1: self.num_cells - 1]
 
             # Tri-diagonal matrix solver
             # High level interface to LAPACK routine
             # https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.lapack.dgtsv.html#scipy.linalg.lapack.dgtsv
             dgtsv(_dl, _d, _du, _b, overwrite_b=1)  # TODO: Do we really need lapack just to do a TDMA solution?
 
-            radial_cell[previous_temperature_idx, :] = _b
-            radial_cell[temperature_idx, :] = _b
-
-            if calculate_at_bh_wall:
-                raise ValueError(
-                    "This portion of the code does not currently run with this "
-                    "vectorized radial numerical implementation."
-                )
```

Comment:
Getting this turned back on is the overall gist of this PR. This adds back computing the STS g-functions when referenced from the borehole wall.